### DISCUSSION TITLE
Using goenv in combination with direnv

### DISCUSSION THREAD

**Discussion by Folaht** [ankitcharolia/goenv#20] [Ideas]
As you know from my [complaint](https://github.com/ankitcharolia/goenv/issues/19),  
I've been trying to figure out a best way not clutter the .bashrc file.  

Now I've found an app called [direnv](https://github.com/direnv/direnv)
This seems the better solution to me.  

Can goenv be designed to work with direnv?

**Comment by ankitcharolia** [comment]
It'a good idea, we have to evaluate the necessary work or we can implement direnv like solution in goenv code.

Thanks for bringing this up!! :+1:
